django/utils/text.py

```diff
@@ -407,9 +408,9 @@ def unescape_string_literal(s):
         raise ValueError("Not a string literal: %r" % s)
     quote = s[0]
     return s[1:-1].replace(r'\%s' % quote, quote).replace(r'\\', '\\')
-unescape_string_literal = allow_lazy(unescape_string_literal)
 
 
+@keep_lazy_text
```

Comment:
Should this be `@keep_lazy(six.text_type, SafeText)` as elsewhere where `SafeText` was previously accepted?